src/builder.rs

```diff
@@ -266,6 +264,26 @@ impl<'req, 'win, W: HasRawWindowHandle> PixelsBuilder<'req, 'win, W> {
     }
 }
 
+pub(crate) fn create_swap_chain(
+    device: &mut wgpu::Device,
+    surface: &wgpu::Surface,
+    format: wgpu::TextureFormat,
+    width: u32,
+    height: u32,
```

Comment:
nitpick: This could accept a `SurfaceSize` as a single arg. Both call sites have direct access to one.

Reply:
Good idea, done.